decidim-core/app/cells/decidim/data_consent/category.erb

```diff
@@ -1,64 +1,67 @@
-<div class="category-wrapper margin-vertical-1" data-id="<%= category[:slug] %>">
-  <div class="category-row flex-center">
-    <button class="dc-title padding-left-3">
-      <span class="h5 dc-category-title">
-        <strong><%= category[:title] %></strong>
-      </span>
-    </button>
-
-    <div class="dc-switch">
-      <input
-        class="switch-input"
-        <%= %(checked="checked") if category[:mandatory] %>
-        id="dc-<%= category[:slug] %>"
-        type="checkbox"
-        name="<%= category[:slug] %>"
-        <%= "disabled" if category[:mandatory] %>>
-
-      <label class="switch-paddle" for="dc-<%= category[:slug] %>">
-        <span class="show-for-sr"><%= t("layouts.decidim.data_consent.modal.toggle", consent_category: category[:title]) %></span>
+<div class="cookies__category" data-id="<%= category[:slug] %>">
```

Comment:
`data-id` is very generic and not very verbose. Wouldn't you prefer `data-category-id` at least?

Reply:
I just left what was initially there... I'll seek where the references are in order to apply more verbosity

Reply:
OK, leave it as it was